packages/components/src/disabled/stories/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Internal dependencies
+ */
+import Disabled from '../';
+import SelectControl from '../../select-control/';
+import TextControl from '../../text-control/';
+import TextareaControl from '../../textarea-control/';
+import CheckboxControl from '../../checkbox-control/';
+import RadioControl from '../../radio-control/';
+import RangeControl from '../../range-control/';
+import ToggleControl from '../../toggle-control/';
+import FormFileUpload from '../../form-file-upload/';
+import AnglePickerControl from '../../angle-picker-control/';
+
+export default {
+	title: 'Components/Disabled',
+	component: Disabled,
+};
+
+export const _default = () => {
+	return (
+		<Disabled>
```

Comment:
Do we need all those components to include in here? Can we use the most representative 3? I don't have answers, I'm thinking whether it's enough to cover those that use `input`, `select` and `textarea`.

Reply:
I questioned that too, and nearly added just a single `input`. I would be fine with making it just the top three, maybe @ItsJonQ might have an opinion?

Reply:
Hai! 👋 
I think max 3 would be okay. Let's go with that 👍

Reply:
Updated :)